phpstan-baseline.neon

```diff
@@ -280,11 +280,6 @@ parameters:
 			count: 3
 			path: eZ/Publish/API/Repository/Tests/SearchServiceTranslationLanguageFallbackTest.php
 
-		-
-			message: "#^Undefined variable\\: \\$loader$#"
-			count: 3
-			path: eZ/Publish/API/Repository/Tests/SetupFactory/Legacy.php
-
```

Comment:
It doesn't seem to be related to the PR. Is this something that fails on 1.3 ATM? Quick check on PHP 8.1 doesn't show it.

Reply:
@alongosz it's not related, but I'm still getting it on PHP 8.1 and PHPStan 1.10.59:
![image](https://github.com/ezsystems/ezplatform-kernel/assets/22300504/8b8d945c-596e-46ac-9f8a-ed8203018872)

Reply:
@Steveb-p this one is particularly weird for me. When you have time could you take a look? For me, also on PHP 8.1 and PHPStan 1.10.59 locally there's no issue neither on `1.3` nor on this (`ibx-6592-remove-location-limitations-object-state`) branch. Logic suggests that for one of the branches it should fail, even if it's about different local environments. Maybe I'm missing something.

Reply:
These lines have been removed in a different PR that went into 1.3 branch already. Because the changes are identical, they are not reported by git as conflicts. Therefore:
1. Original `1.3` branch contained those issues
2. PR was opened against that branch and subsequently tries to remove them.
3. Current `1.3` does not contain them.

I'm rebasing and force pushing the branch.